docs/DeveloperGuide.md

```diff
@@ -308,6 +308,67 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+System: Contact and appointment tracker (MediMeet)
```

Comment:
Can remove this line, it's already mentioned as the default as stated above.

Reply:
Done